### ISSUE TITLE
Autolathe and Circuit Imprinter additions

### ISSUE THREAD

**Issue by DTraitor** [unitystation/unitystation#5576]
### Description

**Autolathe:**
- Add more crafting recipes to autolathe like: qunatum key, wires, guns, etc
- Add access control to autolathe so you cant create guns being assistant but if you use HoS ID on it there will be recipes of guns.
- Allow people emag it

**Circuit imprinter:**
- Add more crafting recipes to it like: chem dispenser board, junkbox board
- Add access control same as autolathe
- Allow people emag it

**Comment by wraith-54321** [comment]
I am guessing this is for until science gets added?

**Comment by DTraitor** [comment]
> I am guessing this is for until science gets added?

Yes and no. On /tg/ you can emag atuolathe to craft some guns and illegal stuff, but it this point of Unitystation you cant do that, but evrything else will be useful until science will be added

**Comment by Fogapod** [comment]
So HoS should make guns himself? Or engineer must have HoS access on his id? Is this how it's done on tg?
Maybe it could accept some kind of permit to make x guns that is destroyed after usage?
Or of you don't want to destroy permits, there could be some item, key, owned by security that allows crafting guns when inserted into autolathe.

**Comment by DTraitor** [comment]
> So HoS should make guns himself? Or engineer must have HoS access on his id? Is this how it's done on tg?
> Maybe it could accept some kind of permit to make x guns that is destroyed after usage?
> Or of you don't want to destroy permits, there could be some item, key, owned by security that allows crafting guns when inserted into autolathe.

I would prefer this situation: Sec need guns so they came to science or eng and ask them to make some on autolathe. HoS use his ID on autolathe and engineer is able to create guns for 5 minutes, after this time HoS should use his ID again to create more

**Comment by PetMudstone** [comment]
Could also add departmental techfabs and other production machines. That way chem dispenser boards will only appear in the medical techfab and guns (and ammo) would only appear in the security techfab.

**Comment by Fogapod** [comment]
Why 5 minutes?

**Comment by DTraitor** [comment]
> Why 5 minutes?

Why not? Eng shouldn't be able to craft shit using autolathe after producing one magazine for sec

**Comment by Fogapod** [comment]
> Eng shouldn't be able to craft shit using autolathe after producing one magazine for sec

So 5 minutes timeout and it's automatically revoked after crafting 1st item? What if sec needs multiple magazines? Unlock autolathe many times? Sounds a bit annoying

**Comment by DTraitor** [comment]
> > Eng shouldn't be able to craft shit using autolathe after producing one magazine for sec
> 
> So 5 minutes timeout and it's automatically revoked after crafting 1st item? What if sec needs multiple magazines? Unlock autolathe many times? Sounds a bit annoying

You can craft as much shit as you need in this 5 minutes

**Comment by Fogapod** [comment]
Don't quite like these arbitrary 5 minutes. It will make sec stay in engineering until timer runs out. Key solution sounds cleaner, also 1 more thing to steal, fun

**Comment by DTraitor** [comment]
> Don't quite like these arbitrary 5 minutes. It will make sec stay in engineering until timer runs out. Key solution sounds cleaner, also 1 more thing to steal, fun

Got new idea. What about same system as security record console has? When you can use your ID to log in, then take it and still be able to use console until you press log out
